### DISCUSSION TITLE
GPS Speed Trigger -- End speed issue with Hero11 Black

### DISCUSSION THREAD

**Discussion by breflick** [gopro/labs#419] [Q&A]
Selected start speed works to trigger start of video recording but recording does not stop at selected end speed.  Anyone else having similar issue?

**Comment by breflick** [comment]
Updating original post from yesterday.

Issue reproducible as same outcome observed during use last evening.

**Reply by dnewman-gpsw** [reply]
This this unrelated, as the other post is about HERO10. 

As the issue is unclear, please film the error as you see it and share that.

**Comment by breflick** [comment]
Speed tab on iPhone QR app is selected.  Start speed and end speed values are entered with start speed at a lower value than end speed.  Generated QR code is successfully scanned by GoPro as check mark is observed followed by speed parameters appearing on screen of GoPro.  GoPro begins recoding video at selected start speed.  Video recording does not subsequently stop at selected end speed regardless of whether or not any hold time is added.

I clarified that other post is about what is occurring with GoPro Hero11 Black.

**Comment by dnewman-gpsw** [comment]
If you start at 20kph and end at 30kph, that is not how the control was designed.  !SK20-30   This tests >=20kph to start and <=30kph to end. So if the initial speed is 25kpm, the camera will start and then immediately stop. 

It was design for a lower end speed than start.    If you want more complex logic, you will need to use the new scripting commands.

In my testing just now, works as designed.

**Reply by breflick** [reply]
Okay.  This is good to know; however, my experience has been the camera does not stop recording once started at start speed instead of stopping immediately. I will hopefully have the opportunity tonight to try settings where end speed is lower than start speed.

**Reply by breflick** [reply]
Changing to have end speed lower than start speed did not resolve issue of recording not pausing (repeat is selected).

**Reply by dnewman-gpsw** [reply]
It works here, and nothing has changed on this feature since it's induction on HERO8. So I recommend you film not working, maybe it will show the cause.  Or do the normal factory reset
and Labs reset.

**Reply by breflick** [reply]
Performed factory reset but not finding clear information about Labs reset.  Saw thread with !RESET QR code as related to Labs reset.  Scanned followed by checkmark then prompted with "three button process" with timed limit.  Didn't execute three button process within time so then "not approved" appear on GoPro screen.  Is !RESET the Labs reset you're referring to?

**Reply by dnewman-gpsw** [reply]
Yes. Press the buttons quickly.

**Reply by breflick** [reply]
Which buttons?  Power On/Off, Video Start/Stop?

**Reply by dnewman-gpsw** [reply]
any you have.

**Comment by breflick** [comment]
Both?  Any order needed?

**Reply by dnewman-gpsw** [reply]
I just press the shutter, but it is like the "Any key" prompt on a computer.

**Reply by breflick** [reply]
Got it.  Done.

**Comment by breflick** [comment]
Another quick question -- Where is GPS time synch function in QR app? Is it the box accompanied by text "Set Date and Time" on left side of QR code?

**Reply by dnewman-gpsw** [reply]
No, that just Sets Date and Time from your phone.  See SYNC under the Extras tab.

**Comment by breflick** [comment]
Got it.  Thanks much!

**Comment by breflick** [comment]
Performed Factory Reset on GoPro and !Reset on Labs.  Scanned GPS speed set for 40kph start and 15kph end.  Device started at 40kph but continues to not stop at selected end speed.

**Comment by breflick** [comment]
Note: Factory Reset and !Reset was repeated since GoPro would not sense GPS speed after initial Factory Reset and !Reset as noted earlier in this thread.

**Comment by dnewman-gpsw** [comment]
I found the issue, and the workaround.  If you use the command !MQRDR=1 , which enables qrcode decoding during recording, this also fixes the issue with the speed trigger.  I tend to have this enabled on all my cameras, so wasn't seeing the issue.   It will be fixed correctly in the next Labs firmware.

**Comment by breflick** [comment]
Thanks.  Looking forward to seeing this work and having fix in next Labs firmware update.

**Reply by breflick** [reply]
Feature now works like a charm.

**Comment by breflick** [comment]
Use of the command !MQRDR=1 indeed fixed the issue.
